examples/rules/data/stories.md

```diff
@@ -1,109 +0,0 @@
-<!-- each story starting with `>>` will be perceived as independent rule -->
```

Comment:
do we still support `md` format? let's not remove that. E.g. for me it is much better format for stories

Reply:
time for change 😆 I can't put both, yaml and md, in there as then everything would bee duplicate. I'm also not sure if we should introduce new features to the old training data format.

Reply:
we should

Reply:
I'm planning to use md format

Reply:
let's just put in a different folder `stories_md` or smth